opty/direct_collocation.py

```diff
@@ -151,6 +151,8 @@ def __init__(self, obj, obj_grad, equations_of_motion, state_symbols,
             ``{x(t): (-1.0, 5.0)}``.
 
         """
+        # needed for the plot_constraint_violations method below
+        self.zeit = node_time_interval
```

Comment:
This should be stored already on ` self.collocator.node_time_interval`.

Reply:
> This should be stored already on ` self.collocator.node_time_interval`.

I did not see this, I will change it.